### ISSUE TITLE
inim -s=mybug.nim crashes (should not crash, as done with `inim ; import mybug`)

### ISSUE THREAD

**Issue by timotheecour** [AndreiRegiani/INim#29]
bugs/t59_array_index.nim:
```nim
let a = @["a", "b", "c"]
echo a
echo a[^0]
```

 * `inim -s=bugs/t59_array_index.nim` should not crash
BAD:
```
inim -s=bugs/t59_array_index.nim
inim_1531876765
/Users/timothee/git_clone/nim/Nim/lib/system.nim(3597) []
/Users/timothee/git_clone/nim/Nim/lib/system.nim(2807) sysFatal
Error: unhandled exception: index out of bounds [IndexError]
Error: execution of an external program failed: '/private/tmp/inim_1531876765 '
```

GOOD: doesn't crash, handles error gracefully when calling a crashing cmd
```
inim
INim 0.2.5
Nim Compiler Version 0.18.1 [MacOSX: amd64] at /Users/timothee/.nimble/bin/nim
>>> import bugs/t59_array_index
t59_array_index
/Users/timothee/git_clone/nim/Nim/lib/system.nim(3597) []
/Users/timothee/git_clone/nim/Nim/lib/system.nim(2807) sysFatal
Error: unhandled exception: index out of bounds [IndexError]
Error: execution of an external program failed: '/private/tmp/inim_1531876771 '
>>>

```

**Comment by AndreiRegiani** [comment]
Fixed in `v0.3.1`

```
# ...
inim> echo a[^0]
Error: unhandled exception: index out of bounds [IndexError]
inim> 123
123 : int literal(123)
```
Doesn't crash / doesn't mess with the compilable code buffer afterwards. Btw you don't need to `echo` these, the discard is feature is working on most cases.

**Comment by timotheecour** [comment]
> Btw you don't need to echo these, the discard is feature is working on most cases.

I do because I'm using `import bugs/t59_array_index` or `inim -s=bugs/t59_array_index.nim` in this case ; in any case moved the discussion to https://github.com/AndreiRegiani/INim/issues/34 (sorry for lack of clarity in the issue that was here)
